src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -952,7 +952,7 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: SlickDataSo
     dataSource.inTransaction { dataAccess =>
       withWorkspaceContextAndPermissions(workspaceName, WorkspaceAccessLevels.Read, dataAccess) { workspaceContext =>
         dataAccess.submissionQuery.listWithSubmitter(workspaceContext).map { _.map {
-            case (submission, user) => new SubmissionStatusResponse(submission, user) 
+            case (submission, user) => new SubmissionListResponse(submission, user)
```

Comment:
If `listWithSubmitter` returned a `ReadAction[Seq[SubmissionListResponse]]` directly, this map step would not be necessary.